packages/ag-charts-types/src/integratedCharts.ts

```diff
@@ -97,7 +97,7 @@ export interface IntegratedModule {
     create(options: AgChartInstanceOptions): AgChartInstance<AgChartInstanceOptions>;
     createSparkline(options: AgSparklineOptions): AgChartInstance<AgSparklineOptions>;
     setup(): void;
-    setGridContext(gridContext: boolean): void;
-    setLicenseKey(licenseKey: string): void;
+    setGridContext?(gridContext: boolean): void;
+    setLicenseKey?(licenseKey: string): void;
```

Comment:
It was Stephen who asked for a consistent API between community and enterprise.
I personally think it's fine, but maybe worth checking that it's consistent with the grid's approach.

Reply:
https://github.com/ag-grid/ag-grid/pull/9292/files

Reply:
I've requested review from @StephenCooper